packages/calypso-build/webpack/sass.js

```diff
@@ -18,6 +18,29 @@ module.exports.loader = ( { includePaths, prelude, postCssOptions } ) => ( {
 			loader: require.resolve( 'css-loader' ),
 			options: {
 				importLoaders: 2,
+				modules: {
+					auto: /\.module\.scss$/, // Only enable CSS modules for .module.scss files
+					getLocalIdent: ( context, localIdentName, localName ) => {
```

Comment:
What are we trying to achieve with this behavior?

Reply:
This is needed for deterministic class name generation between the server and the client.

Reply:
> This is needed for deterministic class name generation between the server and the client, since each Webpack entry processes its own chunk.

Interesting. That makes sense to me, though I would have thought the default behavior would produce a deterministic identifier as well.

One of the concerns I had is whether this could enable someone to continue targeting external CSS styles at the generated name, and if that's something we'd want to be discouraging.

e.g.

```css
[class^="StepContainerV2_style-module-scss__"] {
  // ...some override
}
```

Since the default behavior for CSS module class name generation doesn't have anything static to target, it would not be possible.